assets/js/theme/common/product-details.js

```diff
@@ -56,6 +56,29 @@ export default class ProductDetails {
         this.previewModal = modalFactory('#previewModal')[0];
     }
 
+    /**
+     * https://stackoverflow.com/questions/49672992/ajax-request-fails-when-sending-formdata-including-empty-file-input-in-safari
+     * Safari browser with jquery 3.3.1 has an issue uploading empty file parameters. This function removes any empty files from the form params
+     * @param form: FormData object
+     * @returns FormData object
+     */
+    filterEmptyFilesFromForm(form) {
+        try {
+            for (const pair of form.entries()) {
```

Comment:
🍹 could use destructuring (i.e. `const [key, val] of form.entries()`)

Reply:
do we have browser support requirements?

Reply:
we do but don't need to worry about them in this case.  we've got webpack and babel to transpile down to the envs we specify in the browserlist file.  :)